packages/forms-web-app/src/validators/appeal-householder-decision/enter-code.js

```diff
@@ -0,0 +1,19 @@
+const { body } = require('express-validator');
```

Comment:
couldn't this be made generic for both full-appeal and appeal-householder-decision? (rather than duplicating?)